### ISSUE TITLE
Can I make use of an existing appsetting.development file for local dev over the env in the yaml?

### ISSUE THREAD

**Issue by chivandikwa** [dotnet/tye#1310]
I already have an appsettings.development file in use and would like to use that over setting via env in the yaml for the local development. Is this possible?

**Comment by davidfowl** [comment]
YES but ASP.NET Core doesn't read `ASPNETCORE_ENVIRONMENT` from appsettings.json

**Comment by chivandikwa** [comment]
Thanks for responding @davidfowl just to clarify I set ASPNETCORE_ENVIRONMENT as an environment variable not in the appsettings. Given I have set that to development I expected it would not honor my appsettings.development.json file which it seems it tries by not from the root folder of the solution, not from the project folder for the specific service being targeted. Basically I could not get this to work.

**Comment by davidfowl** [comment]
Ah, its likely because the working directory isn't right? What is the content root of the application? (we print it at startup)
